brandonrhayes/Minimizing_Vaccination_Waste
# Refine tabbing behaviour of GUI, field updates, and error alerts set-up

What's changed:

- Removed "expected demand" input from GUI
- Removed "time to terminate service" input from GUI
- Added "days remaining in replenishment cycle" to GUI
- Updated copy from "Dose type" to "Vial type" and "Number of doses" to "Number of vials"
- Fixed tabbing behaviour to tab in order
- Created an alert to show if first or second dropdown are not selected > Chloe to complete the input validation

How to test:

- Validate existing functionality still works
- Test that alert shows when first/second dropdown not selected
- Test tabbing is in order

Optional:

- Confirm replenishment cycle copy makes sense

Thanks team!
